bin/load-mocks

```diff
@@ -197,7 +198,7 @@ def create_sample_time_series(event, release=None):
     if event is None:
         return
 
-    Event.objects.bind_nodes([event], 'data')
+    eventstore.bind_nodes([event])
```

Comment:
Now that we start using this method, I think we should take a step back and figure out how it will work for any fields that is not 'data'. 'data' fetches everything from nodestore and now it is only used for that :
1) How will this be called for other fields? Is this actually going to support other fields? How is it going to work for nested fields ?
2) Are we going to expose where the fields are fetched from ? Nodestore vs Snuba
3) Are we going to expose which fields have been fetched and which ones have not ?